woocommerce/woocommerce
# fix: storybook product editor webpack path

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Storybook build was broken after https://github.com/woocommerce/woocommerce/pull/49647, this PR fixes it



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. cd into ./tools/storybook
2. run pnpm storybook
3. should work
4. kill process, cd to ./plugins/woocommerce
5. run `pnpm run build` or some variation on it such as `pnpm run watch:build`, and it should work

Note that I'm not 100% sure how to determine in the frontend that `packages/js/product-editor/build/blocks` has been correctly copied over, but using https://github.com/woocommerce/woocommerce/pull/37318 's testing instructions seem to indicate that the directory contents are copied over correctly



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Fixed storybook build 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
